redirector.k8s.io/README.md

```diff
@@ -0,0 +1,36 @@
+Overview
+====
+This contains the TXTDirect configuration for redirector.k8s.io backend service.
```

Comment:
Can you include a brief overview of how it works?  E.g. why does it need a DNS server running in the pod?

Reply:
Will do.

Sidenote: DNS server is not strictly necessary, but helps with latency due to local cache vs Google DNS cache. Also gives us more insight into requested DNS records etc. so generally recommended.